byway/src/main/java/com/google/sps/data/Trip.java

```diff
@@ -37,7 +38,7 @@
   private final String start;
   private final ArrayList<String> destinations;
   private final ArrayList<String> interests;
-  private final ArrayList<String> route;
+  private ArrayList<String> route;
```

Comment:
I know Rena and I also updated the Trip/UserInfo classes for our current PRs. Might be best to update this PR to include all the changes from our PRs and any other changes

Reply:
I think it'll be easier if we just merge what's in master as opposed to looking through branches and adding code you don't really need for your specific PR.